Register.h

```diff
@@ -387,6 +387,624 @@ class RegList0 : public RegisterList<Register> {
   bool buzzerEnabled () const { return this->readBit(DREG_BUZZER_ENABLED,0,false); }
 };
 
+template <class Register>
+class RegList2 : public RegisterList<Register> {
+public:
+  RegList2 (uint16_t addr) : RegisterList<Register>(addr) {}
+  bool    ADDRESS_SENDER1_HIGH_BYTE    (uint8_t value) const { return this->writeRegister(1  , value & 0xff); }

```

Comment:
It may be better to use index as argument
```
bool ADDRESS_SENDER_HIGH_BYTE(uint8_t index,uint8_t value) const { return this->writeRegister(1+(index*7),value); }
uint8_t ADDRESS_SENDER_HIGH_BYTE(uint8_t index) const { return this->readRegister (1+(index*7), 0); }
```
.....

Reply:
> It may be better to use index as argument

I thought it is more inuitive to use the decimal index values directly as they were used in the device description xml file:
https://github.com/AskSinPP/asksinpp.de/blob/master/rftypes/xmls/rf_rep.xml#L68

But only for the register, it's not the problem to leave this in the sketch.

Reply:
Changes made and pushed